src/pages/comparison/comparison.tsx

```diff
@@ -613,6 +733,11 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
               onClose={() =>
                 handleChangeRightPanelVisibility(ActiveButton.options)
               }
+              baseMaps={baseMaps}
+              selectedBaseMapId={selectedBaseMap.id}
+              insertBaseMap={handleInsertBaseMap}
```

Comment:
Nit: there is inconsistency in naming between base map callbacks and layer callbacks:
eg `onLayerInsert` - `insertBaseMap`. If we think on them as event handlers I like `on` prefix as it points on callbacks and verb is in the end of name. So `onLayerInsert` is better format for my experience.
On the other hand, order callbacks that are not event handlers should start from verb. in that case `insertBaseMap` is better format.
I like  the "event handler" format in this case and of course these callbacks should align the same naming convention.